frontend/src/component/common/StrategyItemContainer/StrategyItemContainer.tsx

```diff
@@ -16,16 +15,20 @@ type StrategyItemContainerProps = {
     onDragStart?: DragEventHandler<HTMLButtonElement>;
     onDragEnd?: DragEventHandler<HTMLButtonElement>;
     headerItemsRight?: ReactNode;
+    headerItemsLeft?: ReactNode;
     className?: string;
     style?: React.CSSProperties;
     children?: React.ReactNode;
 };
 
-const DragIcon = styled(IconButton)({
+const inlinePadding = 3;
```

Comment:
Offset the drag icon so that the strategy design matches the sketches (strategy title is aligned with strategy content such as constraints.